src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -11,10 +11,10 @@ namespace System.Text.Json
     public sealed partial class Utf8JsonWriter
     {
         /// <summary>
-        /// Writes the pre-encoded property name and raw bytes value (as a base 64 encoded JSON string) as part of a name/value pair of a JSON object.
+        /// Writes the pre-encoded property name and raw bytes value (as a Base64 encoded JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
         /// <param name="propertyName">The JSON-encoded name of the property to write.</param>
-        /// <param name="bytes">The Base64-encoded data to write.</param>
+        /// <param name="bytes">The binary data to write as a Base64 encoded text.</param>
```

Comment:
The 'a' here should be removed, e.g. "as a Base64 encoded text" => "as Base64 encoded text".  Same for other places "Base64 encoded" describes "text".

Reply:
I find it weird without the article.

Reply:
"as a text" isn't grammatically correct, unless you're talking about "text" as something that's singular, e.g. a tome. It should be "as text".

Reply:
Thanks! @rpetrusha just confirmed as well. Sorry about the wrong guidance @safern.

Reply:
FIxed. Thanks, @stephentoub